web/html/src/components/buttons.tsx

```diff
@@ -159,17 +158,23 @@ export type ButtonProps = BaseProps & {
 export class Button extends _ButtonBase<ButtonProps> {
   render() {
     const text = this.props.text ?? this.props.children;
+    const cssClasses = "btn " + (this.props.className ?? "btn-default");
+
     return (
       <button
         id={this.props.id}
         type="button"
         title={this.props.title}
-        className={"btn " + (this.props.className ?? "")}
+        className={cssClasses}
         onClick={this.props.handler}
         disabled={this.props.disabled}
       >
-        {this.renderIcon()}
-        {text}
+        {
```

Comment:
The brackets and the fragment are superfluous here, what was present before works fine:

```
{this.renderIcon()}
{text}
```